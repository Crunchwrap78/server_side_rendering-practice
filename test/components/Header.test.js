import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../src/components/Header';

let wrapper;

test('should render Header correctly', () => {
  wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
