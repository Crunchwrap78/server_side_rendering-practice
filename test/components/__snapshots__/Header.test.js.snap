// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <div
    className="logo"
  >
    Logo
  </div>
  <nav>
    <a
      href="#"
    >
      Create Ads 
    </a>
    <a
      href="#"
    >
      About Us 
    </a>
    <a
      href="#"
    >
      Log In 
    </a>
    <a
      className="register-btn"
      href="#"
    >
      Register
    </a>
  </nav>
</header>
`;
